warrior/Actions/KafkaActions/kafka_actions.py

```diff
@@ -68,12 +69,32 @@ def send_messages(self, system_name, topic, value,
         status = True
         if not data_repository.get("kafka_producer", None):
             print_info("creating kafka producer")
-            kafka_ip = getSystemData(self.datafile, system_name, "ip")
-            kafka_port = getSystemData(self.datafile, system_name, "kafka_port")
-            ca_file = getSystemData(self.datafile, system_name, "ssl_cafile")
-            key_file = getSystemData(self.datafile, system_name, "ssl_keyfile")
-            crl_file = getSystemData(self.datafile, system_name, "ssl_crlfile")
-            ciphers = getSystemData(self.datafile, system_name, "ssl_ciphers")
+            # mapper file will be available for netconf backup
+            map_file = data_repository.get('wt_mapfile', None)
+            if map_file:
+                ca_file = key_file = crl_file= ciphers = None
+                config = ConfigObj(map_file)
+                mapper = config["CREDENTIALS"]["kafka_server"]
```

Comment:
this syntax results in KeyError if CREDENTIALS or kafka_server not present in mapper file.
use like this.
    if "CREDENTIALS" not in config and "kafka_server" not in config["CREDENTIALS"]:
        return False
    mapper = config["CREDENTIALS"]["kafka_server"]
    status, mapper_data = Utils.data_Utils.replace_var(mapper, {}, {})
    return status if not status
    kafka_ip = config["CREDENTIALS"]["kafka_server"].get("kafka_host", None)
    kafka_port = config["CREDENTIALS"]["kafka_server"].get("kafka_port", None)   
    ca_file = config["CREDENTIALS"]["kafka_server"].get("ca_file", None)
    key_file = config["CREDENTIALS"]["kafka_server"]("key_file", None)
    crl_file = config["CREDENTIALS"]["kafka_server"]("crl_file", None)
    ciphers = config["CREDENTIALS"]["kafka_server"]("ciphers", None)